src/test/java/ch/wisv/areafiftylan/EventRestIntegrationTest.java

```diff
@@ -0,0 +1,769 @@
+package ch.wisv.areafiftylan;
+
+import ch.wisv.areafiftylan.model.Event;
+import ch.wisv.areafiftylan.model.Team;
+import ch.wisv.areafiftylan.model.User;
+import ch.wisv.areafiftylan.service.repository.EventRepository;
+import ch.wisv.areafiftylan.service.repository.TeamRepository;
+import ch.wisv.areafiftylan.service.repository.UserRepository;
+import ch.wisv.areafiftylan.util.SessionData;
+import com.jayway.restassured.http.ContentType;
+import org.apache.http.HttpStatus;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.*;
+
+
+/**
+ * Created by Sille Kamoen on 27-3-16.
+ */
+public class EventRestIntegrationTest extends IntegrationTest {
+
+    private final String EVENTS_ENDPOINT = "/events";
+    protected Event testEvent;
+
+    private Team team1;
+    private Team team2;
+
+    private User teamCaptain;
+
+    @Autowired
+    EventRepository eventRepository;
+
+    @Autowired
+    TeamRepository teamRepository;
+
+    @Autowired
+    UserRepository userRepository;
+
+    @Before
+    public void setupEventTests() {
+        testEvent = new Event("Test Event", 2, 5);
+        testEvent.setOpenForRegistration(true);
+        testEvent = eventRepository.save(testEvent);
+
+        teamCaptain = userRepository
+                .save(new User("captain", new BCryptPasswordEncoder().encode("password"), "captain@team.com"));
```

Comment:
Please use the new password methodology. We are using a single final string in the tests instead of hardcoded strings.